monero-outreach-docs/translations/es/archive/getting_started_es.md

```diff
@@ -0,0 +1,197 @@
+[Únete a la conversación e involucrate](https://www.monerooutreach.org/stories/getting-started-helping-monero.html#join)
+[Otras formas de ayudar](https://www.monerooutreach.org/stories/getting-started-helping-monero.html#otherways)
+[Monero 101](https://www.monerooutreach.org/stories/getting-started-helping-monero.html#monero101)
+[Minar Monero](https://www.monerooutreach.org/stories/getting-started-helping-monero.html#mining)
+
+### _Desarrolladores_
+
+Si eres un desarrollador, te invitamos a que te unas a algún equipo.
+
+La principal manera de empezar, es familiarizarse con la base del código, el estándar del protocolo cryptonote e interactuar con otros desarrolladores. Todos los desarrolladores se comunican a través de IRC en #monero-dev o #monero en los servidores de freenode. Encuentra aquí recursos para que los desarrolladores puedan empezar:
+
+- Código Fuente - [github.com/monero-project](https://github.com/monero-project)
+- Asuntos(Issues) abiertos - [github.com/monero-project/monero/issues](https://github.com/monero-project/monero/issues)
+- El ecosistema de Monero - [github.com/monero-ecosystem](https://github.com/monero-ecosystem)
+
+- Guías para desarrolladores - [web.getmonero.org/es/resources/developer-guides](https://web.getmonero.org/es/resources/developer-guides/)
+- Apuntes de las reuniones de los desarrolladores - [monerobase.com](https://monerobase.com/)
+- Espacios de desarrollo de Monero - [moneroworld.com/development.html](https://moneroworld.com/development.html)
+- Laboratorio de Investigación de Monero - [web.getmonero.org/es/resources/research-lab](https://web.getmonero.org/es/resources/research-lab/)
+
+Los desarrolladores también podrían estar interesados en participar en alguno de los equipos de trabajo existentes dentro de la comunidad de Monero. Pueden encontrar una lista completa de los más activos más abajo.
+
+### _Equipos de Trabajo_
+
+Abajo encontrarás la lista de los equipo de trabajo de Monero que se actualizaran a medida que nuevos equipos se establezcan. Si tienes una idea de un proyecto para apoyar a Monero que no este en la lista, toma la iniciativa y empieza. Existen algunos recursos disponibles para la comunidad de Monero para ayudar al desarrollo efectivo de los equipos de trabajo y sus operaciones. Si tienen preguntas vayan alguno de los foros y pregunten lo que necesiten. Tenemos decenas de colaboradores que pueden ayudarlos a arrancar.
+
+#### Desarroladores
+Repositorio de desarrollo de Monero, incluidos el daemon, la CLI, GUI y otros software cruciales.
+_¿Dónde empezar?_ [Ir a la sección](https://www.monerooutreach.org/stories/getting-started-helping-monero.html#developers)
+
+#### Comunidad
+Organizar reuniones, proveer materiales útiles y el Coffee chat mensual.
+_¿Dónde empezar?_ [r/MoneroCommunity](https://www.reddit.com/r/MoneroCommunity/)
+
+#### Localización
+Traducción de materiales relacionados con Monero. Si hablan otro idioma, revisen esto.
```

Comment:
I feel like the translation is using a way to speak to the reader in plural, in example here:

_Si hablan otro idioma, revisen esto._

The original file, as per language (English) uses more like a "you", singular way to refer to the reader, not in plural. It may be interpreted different but a text usually talks to you in singular rather than plural. The main point would be:

_Si hablas otro idioma, revisa esto._

It's open to review since you used that way in other parts, it can be just left as is.

Reply:
Yeah, I was trying not to translate in singular but while I was reading it again it reads weird when translated to plural instead of singular. I'm making the changes. Thanks.

Reply:
It's common when translating, but you'll also need to be on the other side of the translation, thanks lh1008.